Carbon.Redis/README.Md

```diff
@@ -0,0 +1,264 @@
+﻿# Carbon.Redis
+
+This package is powered up by multiple useful redis libraries such as Stackexchange.Redis [1], RedLock.net [2], Microsoft.Extensions.Caching.StackExchangeRedis [3]  that provide remarkable Redis capabilities
+to your projects in many aspect.
+
+>[1] https://github.com/StackExchange/StackExchange.Redis
+> 
+>[2] https://github.com/samcook/RedLock.net
+> 
+>[3] https://github.com/dotnet/aspnetcore
+
+This package wraps up some capabilities such as (Redis Lock, Redis PubSub, Redis Key Operations) in a single pack and presents you with an easier and maintained way. It also creates some handy logics for Redis Operations which
+creates ease of usage and out-of-the-box serialization switch enablement which provides you some object conversions on behalf of you and also a Redis Healthcheck will be added
```

Comment:
which provides you some object conversions on behalf of you => which provides you with some object conversions